docs/references/api/deviceshifu-deployment.md

```diff
@@ -72,3 +72,17 @@ For a detailed example, please refer to https://github.com/Edgenesis/shifu/tree/
     - **IP_CAMERA_PASSWORD** (string)<br/>password for the camera RTSP stream.
     - **IP_CAMERA_CONTAINER_PORT**
         port of the RTSP camera driver container, which has to be `11111` (for now).
+
+## LwM2M DeviceShifu Deployment
+
+For a detailed example, please refer to [https://github.com/Edgenesis/shifu/tree/main/examples/lwM2MDeviceShifuWithSecurity](https://github.com/Edgenesis/shifu/tree/main/examples/lwM2MDeviceShifuWithSecurity).
+
+- **spec.template.spec.containers[0].image**
+  - **name** (string)<br/>driver image for the LwM2M device, the test version of the driver image for the LwM2M device is `edgehub/deviceshifu-http-lwm2m:nightly`. Please use the stable version in the production environment.
+- **spec.template.spec.containers[0].ports[0].containerPort**
+  - **containerPort** (integer)<br/>port for the DeviceShifu HTTP server, set to `8080`.
+- **spec.template.spec.containers[0].env**
+  - **EDGEDEVICE_NAME** (string)<br/>name of the EdgeDevice associated with the DeviceShifu, such as `edgedevice-lwm2m`.
+  - **EDGEDEVICE_NAMESPACE** (string)<br/>namespace for the EdgeDevice corresponding to DeviceShifu, set to `deviceshifu`.
+  - **LOG_LEVEL** (string)<br/>log level for the EdgeDevice corresponding to DeviceShifu, such as  `debug`.
+  - **LWM2M_ENDPOINT** (string)<br/>endpoint of the LwM2M client, such as `coap://127.0.0.1:5683`.
```

Comment:
remove this two field, log level is only for develop and LwM2M Endpoint is not exists